frontend/src/App.tsx

```diff
@@ -26,16 +44,42 @@ const router = createHashRouter([
   },
 ])
 
+const VITE_NETWORK_NUMBER = Number(VITE_NETWORK)
+
+export const wagmiConfig = createConfig({
+  // multiInjectedProviderDiscovery: false,
```

Comment:
Note to self: Need to explore why this should be disabled in the docs.

Reply:
@matevz this issue might brake mobile support, due to current implementation of wagmi-v2 wrapper.